CHANGELOG.md

```diff
@@ -1,8 +1,55 @@
-# master
+# v0.13.1: Meta tag and http api improvements, lineage metadata, per partition metrics and more. Nov 28, 2019.
+
+## meta tags
+
+* correctly clear enrichment cache on upsert #1472
+* meta tag records must be optional in meta tag upsert requests #1473
+* Persist meta records to Cassandra index #1471
+* monitor efficiency of enrichment cache #1511
```

Comment:
not sure if this still needs to be here, because #1523 has completely removed the enrichment cache

Reply:
thx. PTAL